lib/rules/style/sheet.js

```diff
@@ -25,7 +25,7 @@ exports.check = function (sr, done) {
     const lnk = sr.jsDocument.querySelectorAll(stylesheetLinks.join(', '));
     if (!lnk.length) sr.error(missing, 'not-found', { url });
     else {
-        const siblings = util.nextAll(lnk, 'link[rel=stylesheet], style');
+        const siblings = util.nextAll(lnk[0], 'link[rel=stylesheet], style');
```

Comment:
Does this mean that rule was always skipped?

Reply:
Yes, there's was a bug here